### DISCUSSION TITLE
1.3 release?

### DISCUSSION THREAD

**Discussion by avee87** [cioraneanu/firefly-pico#91] [General]
It's been a while since the last actual release. There have been a couple of important features and fixes since 1.2 and it would be nice to have a non-dev version incorporating them.

Any estimates when 1.3 may happen? What would be the criteria for shipping it?

**Comment by cioraneanu** [comment]
Version 1.3 will be ready in about 1 week. My initial target was to have a big release every month but of course varies based on the available time. 
The current criteria is to wait until there several big new features in each release. Later when the ideas pool get smaller I'm ok with shipping smaller changes.
